plugins/foreman_ansible/1.x/index.md

```diff
@@ -84,11 +84,28 @@ Run ansible on your hosts and the callback will send everything to Foreman autom
 
 ![registering a host](static/images/plugins/foreman_ansible/registering_a_host.gif)
 
-If you want to assign Ansible roles to your hosts, please put them on `/etc/ansible/roles/`. After that the plugin will detect them and you will be able to assign them to your hosts when you edit them.
+## 4.1 Importing Roles
 
-![adding roles](static/images/plugins/foreman_ansible/adding_roles.png)
+Ansible roles can be imported from a smart proxy that has 'Ansible' feature or from your local '/etc/ansible/hosts'. List of all roles already imported into Foreman can be accessed through 'Configure' in the main menu. You can select the source for import from the dropdown on the right.
```

Comment:
`/etc/ansible/roles` not `hosts`. Instead of from your 'local' , I'd specify from your 'Foreman host', it's less ambiguous.

`You can import hosts from the dropdown` - and add a screenshot  

typos
- 'List of all roles` -> `A list of all roles`